CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the code was deployed.
 ### Changed
 - Update landing page to display list of locations with how long ago most recent session was started (CU-mmzgy9)
 - Update checkHeartbeat to check both XeThru and INS (CU-m0wdfn)
+- Using https on backend (CU-mt3fah)
+- Updating brave-alert-lib to v2.4.0 (CU-mt3fah)
```

Comment:
This update includes the thing where I removed the timestamp from `helpers.log`. So we should also turn on Kubernetes logging at the same time that this is merged.

Reply:
good note!

Reply:
Is this something that is changed in the helm config and would be a part of this PR or will this happen elsewhere?

Reply:
so this is actually already enabled by default in kubernetes, you can choose whether to display them with or without. For example here's how it looks now: 
![image](https://user-images.githubusercontent.com/7051376/112889714-dcd89480-908a-11eb-8f7f-6b404d50320d.png)

previously it would double up the timestamp

Reply:
Sweet. I didn't know about `--timestamps`. Might be worth putting in the README. Thanks! I don't know why we wouldn't want them, eh?